### ISSUE TITLE
AttributeError: `np.float_` was removed in the NumPy 2.0 release. Use `np.float64` instead.

### ISSUE THREAD

**Issue by Blanket58** [aliyun/aliyun-odps-python-sdk#239]
In `odps.types` line 1557

```python
integer_builtins = six.integer_types
float_builtins = (float,)
try:
    import numpy as np
    integer_builtins += (np.integer,)
    float_builtins += (np.float_,)
except ImportError:
    pass
```

**Comment by wjsi** [comment]
Will resolve in next release.

**Comment by Smallhi** [comment]
+1

**Comment by wjsi** [comment]
Fixed with v0.11.6.2.
